pyproject.toml

```diff
@@ -7,3 +7,9 @@ force-exclude = '''
   | tests/protos
 )/
 '''
+
+[tool.pytest.ini_options]
+log_cli = true
+log_cli_level = "INFO"
+log_cli_format = "%(asctime)s [%(levelname)8s] %(message)s (%(filename)s:%(lineno)s)"
+log_cli_date_format = "%Y-%m-%d %H:%M:%S"
```

Comment:
```suggestion
[tool.pytest.ini_options]
log_cli = true
log_cli_level = "INFO"
log_cli_format = "%(asctime)s [%(levelname)8s] %(message)s (%(filename)s:%(lineno)s)"
log_cli_date_format = "%Y-%m-%d %H:%M:%S"
```

I don't think we need live logs because they make it difficult to check pytest logs.

```
tests/xgboost/test_xgboost_model_export.py::test_pyfunc_serve_and_score 
--------------------------------------------------- live log call ----------------------------------------------------
INFO     89d4b8295536_create_latest_metrics_table_py:89d4b8295536_create_latest_metrics_table.py:153 Migration complete!
INFO     2b4d017a5e9b_add_model_registry_tables_to_db_py:2b4d017a5e9b_add_model_registry_tables_to_db.py:46 Adding registered_models and model_versions tables to database.
INFO     2b4d017a5e9b_add_model_registry_tables_to_db_py:2b4d017a5e9b_add_model_registry_tables_to_db.py:77 Migration complete!
FAILED                                                                                                         [ 33%]
tests/xgboost/test_xgboost_model_export.py::test_pyfunc_serve_and_score_sklearn[model0] 
--------------------------------------------------- live log call ----------------------------------------------------
INFO     89d4b8295536_create_latest_metrics_table_py:89d4b8295536_create_latest_metrics_table.py:153 Migration complete!
INFO     2b4d017a5e9b_add_model_registry_tables_to_db_py:2b4d017a5e9b_add_model_registry_tables_to_db.py:46 Adding registered_models and model_versions tables to database.
INFO     2b4d017a5e9b_add_model_registry_tables_to_db_py:2b4d017a5e9b_add_model_registry_tables_to_db.py:77 Migration complete!
PASSED                                                                                                         [ 66%]
tests/xgboost/test_xgboost_model_export.py::test_pyfunc_serve_and_score_sklearn[model1] 
--------------------------------------------------- live log call -----------------------
```

Reply:
We can view logging messages when a test fails. If we need to view logging messages for successful tests, we can use `-s` which disables all capturing.

Reply:
Done!